var/spack/repos/builtin/packages/libconfig/package.py

```diff
@@ -16,7 +16,10 @@ class Libconfig(AutotoolsPackage):
     # there is currently a build error with version 1.6, see:
     # https://github.com/hyperrealm/libconfig/issues/47
     # version('1.6', '2ccd24b6a2ee39f7ff8a3badfafb6539')
-    version('1.5', 'e92a91c2ddf3bf77bea0f5ed7f09e492', preferred=True)
+    version('1.7.1',
+        sha256='d288e6ae817f4ef78df43cdb2647f768dc97899ee82fcc41f857e8eb9fd7fbdb',
+        preferred=True)
```

Comment:
Can you move 1.7.1 above 1.6? Also, you can remove `preferred=True` since the latest stable release is always preferred.

Reply:
Thank you.  The changes are in https://github.com/spack/spack/pull/12324/commits/2efaa73da13f0a49545412f5c3b7ae86ec990638 .